### PR TITLE
introduce long-lived HNL->ee as example analysis

### PR DESCRIPTION
**Pull request by jalimena**
using FCCAnalysisRun for all steps of the analysis chain:

1. analysis_stage1.py
2. analysis_final.py
3. analysis_plots.py

as requested (https://github.com/HEP-FCC/FCCAnalyses/pull/157#issuecomment-1116046432)

### CODE REVIEW

File: examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+prodTag     = "FCCee/spring2021/IDEA/"
+
+#Optional: output directory, default is local dir
+outputDir = "/eos/experiment/fcc/ee/analyses/case-studies/bsm/LLPs/HNLs/"
+#outputDir = "/eos/user/j/jalimena/FCCeeLLP/"
+#outputDir = "output_stage1/"
+
+outputDirEos = "/eos/experiment/fcc/ee/analyses/case-studies/bsm/LLPs/HNLs/"
+#outputDirEos = "/eos/user/j/jalimena/FCCeeLLP/"
+eosType = "eosuser"
```
Comment (clementhelsens) [inline examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py]: eosuser is not the flavour of fcc eos, please remove

**Comment by clementhelsens** [comment]
Thanks for adding this example @jalimena, does this example need the other developments we did about using custom code?

**Comment by jalimena** [comment]
hi @clementhelsens , somehow actually no, I found that in this configuration, the custom batch config isn't needed. I guess all your other developments made that unnecessary.

File: examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py (hunk 1 above)
Comment (clementhelsens) [reply examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py]: Hi @jalimena , I'd like to merge it but I think the two eos things are not compatible, could you check please?

**Comment by jalimena** [comment]
reminder @clementhelsens

**Comment by clementhelsens** [comment]
@jalimena , yes, I remember, please see my pending comment inline

File: examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py (hunk 1 above)
Comment (jalimena) [reply examples/FCCee/bsm/LLPs/DisplacedHNL/analysis_stage1.py]: should be now fixed, thanks!

**Pull request merged**